### PR TITLE
Issue 14 - Add product categories models

### LINKED ISSUE #14: Add product categories models
Categories should have:
- id (auto field)
- name
- parent (foreign key to self or None)
- active (if inactive should not be represented on fetch etc)
- deleted (if deleted - the same as active=False and readonly from now on)

Admin view should be added
API Resource should be added here (public w/o access restrictions with all common methods)

### PR DESCRIPTION
**Pull request by sshishov**
`ProductCategory` model is added
`ProductCategory` admin view is registered
`ProductCategoryResource` resource API is added
`StoreBaseResource` is created to keep common functionality across `store` resources

Initial migration has been changed with product category functionality
Additionally `test_store.json` fixture is added to populate test store with initial test data

@zitoss , @Gosha2015  please review

Fixes #14

### CODE REVIEW

**Comment by sshishov** [comment]
This coverage omission will recovered in upcoming written tests

**Comment by zitoss** [comment]
Please create test cases. Need fully to cover the solution by codecov.

**Comment by sshishov** [comment]
Tests have been added

File: avtozip/store/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"""Store __str__ representation of models."""
```
Comment (zitoss) [inline avtozip/store/tests/test_models.py]: Please put clear description for docs comment here

**Comment by zitoss** [comment]
Rest looks good

**Comment by zitoss** [comment]
Perfectly, please set label `rebased` and merge the solution.

**Comment by sshishov** [comment]
I will set `rebased` as soon as I have rebased this PR :laughing:

**Pull request merged**